CHANGELOG.md

```diff
@@ -64,6 +64,7 @@
    * FIXED: Fix Matrix API to return correct end location. [#5509](https://github.com/valhalla/valhalla/pull/5509)
    * FIXED: Set node snap flags properly in PBF PathEDGE [#5508](https://github.com/valhalla/valhalla/pull/5508)
 * **Enhancement**
+   * CHANGED: Optimise Turn::GetType using lookup table [#5530] (https://github.com/valhalla/valhalla/pull/5530)
```

Comment:
we keep an ascending changelog, so new entries go last

Reply:
Updated, ptal again :)